components/layout/navbar/navbar.tsx

```diff
@@ -125,18 +124,33 @@ const Navbar = ({ DesHeight }) => {
           font-weight: bold;
         }
 
+        .inner-logo {
+          width: inherit;
+          height: inherit;
+          transform: rotate(45deg);
+          transition: transform 0.3s linear;
+        }
+
         .navbar__logo {
-          position: relative;
-          top: 25px;
-          left: -15px;
-          z-index: 1;
-          width: 82px;
-          height: 82px;
+          margin-top: 52px;
```

Comment:
### from here on: 
pretty sure it is not pixel perfect, so let me know if you want me to fix it and how (: